pkg/k8s/k8siface.go

```diff
@@ -70,12 +72,13 @@ func (k *k8sIface) ListGatewayNodes() (*v1.NodeList, error) {
 }
 
 func (k *k8sIface) updateLabel(nodeName string, mutate func(existing *v1.Node)) error {
-	// nolint:wrapcheck // Let the caller wrap these errors.
 	client := &resource.InterfaceFuncs{
 		GetFunc: func(ctx context.Context, name string, options metav1.GetOptions) (runtime.Object, error) {
+			// nolint:wrapcheck // Let the caller wrap these errors.
 			return k.clientSet.CoreV1().Nodes().Get(ctx, name, options)
 		},
 		UpdateFunc: func(ctx context.Context, obj runtime.Object, options metav1.UpdateOptions) (runtime.Object, error) {
+			// nolint:wrapcheck // Let the caller wrap these errors.
```

Comment:
This works but I meant to put the comment back before L75 where it was previously.